src/Google/Client.php

```diff
@@ -1162,6 +1162,9 @@ protected function createDefaultHttpClient()
     } elseif (6 === $guzzleVersion || 7 === $guzzleVersion) {
       // guzzle 6 or 7
       $options['base_uri'] = $this->config['base_path'];
+
+      $options['http_errors'] = $options['exceptions'];
+      unset($options['exceptions']);
```

Comment:
Seems like we should just remove the setting of `exceptions` above and do the following instead for more obvious/straightforward logic:

```php
    if (5 === $guzzleVersion) {
      $options = [
        'base_url' => $this->config['base_path'],
        'defaults' => ['exceptions' => false],
      ];
      if ($this->isAppEngine()) {
        // set StreamHandler on AppEngine by default
        $options['handler'] = new StreamHandler();
        $options['defaults']['verify'] = '/etc/ca-certificates.crt';
      }
    } elseif (6 === $guzzleVersion || 7 === $guzzleVersion) {
      // guzzle 6 or 7
      $options[
          'base_uri' => $this->config['base_path'],
          'http_errors' => false,
      ];
    } else {
      throw new LogicException('Could not find supported version of Guzzle.');
    }
```

Reply:
Done. It makes more sense indeed. Thanks for your feedback.